WordPress/wordcamp.org
# Changes to camptix-invoices plugin

As the Community Team agreed to activate the Camptix Invoices plugin for all new WordCamps by default, I started looking at the plugin's codebase to identify if there are things that need some tweaks to make it work without organisers needing to update settings.

In this PR:
- Enable invoices by default
- Enable VAT field by default
- Use site date format as default in the invoice
- If all enabled payment methods use allowed predefined WPCS accounts, use WPCS address in the invoice. In other cases, default to the site name and ask organisers to update the setting.
- Makes address and VAT fields optional
- Prevents the email being sent if no attachment

Fixes #928, #255, #640
See #971 

Props @javiercasares, @harmonyromo, @ciudadanoB, @dd32 

Things identified that need work:
- [x] Handle invoice sender details when WordCamps is not using WPCS to run money
- [x] Possibly enable VAT field by default
- [x] Activate invoices by default (the setting)
- [x] Add plugin to list of network activated ones
- [x] Make VAT field optional
- [ ] Network activate the plugin

### Screenshots

Camptix invoices settings with new defaults.
<img width="708" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/415544/b68ca2c3-9a72-4ed6-8d41-1f955703953a">

Paypal payment method enabled with custom and Stripe enabled with WPCS, notice about company address.
<img width="909" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/415544/a0f37e6d-7269-4bfa-b658-e70bd58fbad3">

All payment methods using WPCS, company address field hidden from invoice settings.
<img width="669" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/415544/1c71192c-4651-453d-bb6d-54e10e674af0">

Invoice fields on tickets page.
<img width="604" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/415544/2af58eb2-4888-4eb3-9eef-10be61fdcacf">

Invoice PDF.
<img width="867" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/415544/ffcd55b0-6171-4fd8-ab6c-a4a91d240d0e">


### How to test the changes in this Pull Request:
1.  Create new WordCamp site and activate Camptix Invoices
2. Navigate to Camptix settings, invoices tab -> invoices and VAT should be enabled
3. Go to payment tab, activate Paypal or Stripe with custom account -> you should see notice about company address
4. Activate the other payment method with WPCS account -> you should still see the notice
5. Change the first payment method to use WPCS account -> notice should dissapear
6. Go buy ticket, request invoice -> invoice should be sent and WPCS address should be in invoice
